# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../loudmouth/eai_nodata.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./loudmouth/lm-sock.c.orig	2009-11-15 17:40:11.000000000 +0100
+++ ./loudmouth/lm-sock.c	2009-11-15 17:41:33.000000000 +0100
@@ -310,9 +310,11 @@
 	case EAI_FAIL:     
 		return _("The nameserver encountered errors "
 			 "looking up this address");
+#if defined(EAI_NODATA) && (EAI_NODATA != EAI_NONAME)
 	case EAI_NODATA:   
 		return _("The remote host exists but no address "
 			 "is available");
+#endif
 	case EAI_NONAME:   
 		return _("The remote address is unknown");
 	case EAI_FAMILY:
